modules/microshift-accessing-cluster-remotely.adoc

```diff
@@ -43,7 +42,7 @@ The `user@workstation` login is used to access the host machine remotely. The `<
 +
 [NOTE]
 ====
-To generate the `kubeconfig` files for this step, see link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{product-version}/html/configuring/microshift-kubeconfig#generating-additional-kubeconfig-files_microshift-kubeconfig[Generating additional kubeconfig files for remote access].
+To generate the `kubeconfig` files for this step, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.16/html/configuring/microshift-kubeconfig#generating-additional-kubeconfig-files_microshift-kubeconfig[Generating additional kubeconfig files for remote access].
```

Comment:
This link takes you to "Cluster access with kubeconfig" instead of directly to the "Generating additional kubeconfig files for remote access" procedure.